### PR TITLE
Update procedure CIP

### PR DESCRIPTION
**Pull request by Lojjs**

### CODE REVIEW

File: cip/1.accepted/CIP2015-06-24-call-procedures.adoc [hunk 1]
```diff
@@ -137,10 +137,12 @@ In this case, the trailing `RETURN` clause is omitted and all projected fields a
...
-The `YIELD` subclause may only be omitted in the standalone form of `CALL` to call a procedure that does not return any result fields.
-In this case the query will return as many (empty) rows as produced by the called procedure.
+The `YIELD` subclause may be omitted in the standalone form of `CALL`. 
+In this case all result fields produced by the procedure implementation are returned by the statement.
+For procedure that does not return any result fields, the query will return as many (empty) rows as produced by the called procedure.
 
-A further simplification allowed in the standalone form is to use `YIELD *` to denote that all result fields produced by the procedure implementation are returned by the statement.
+Another subclause allowed in the standalone form is `YIELD *`, which denotes that all result fields produced by the procedure implementation are returned by the statement.
+For procedure that does not return any result fields, the query will return as many (empty) rows as produced by the called procedure.
 The `YIELD *` form is _only_ allowed in the standalone form of `CALL`.
```
Comment (Hunterness) [inline cip/1.accepted/CIP2015-06-24-call-procedures.adoc:146]: This might be redundant

File: cip/1.accepted/CIP2015-06-24-call-procedures.adoc [hunk 2]
```diff
@@ -190,31 +192,32 @@ RETURN *
...
-.Standalone call to a procedure that does not return result fields
+.Standalone procedure call with omitted `YIELD` and implicitly passed arguments
 [source, cypher]
 ----
-CALL myProc($arg)
+CALL myProc
 ----
 
-Omitting all result fields when calling a procedure may still be achieved using an in-statement call:
+==== Standalone call with omitted `YIELD` clause
```
Comment (Hunterness) [inline cip/1.accepted/CIP2015-06-24-call-procedures.adoc:213]: This and the section on line 207 feels a bit repetitive, especially directly after each other

Comment (Hunterness) [reply cip/1.accepted/CIP2015-06-24-call-procedures.adoc:213]: This section should be part of standalone call section? Not on the same level?

File: cip/1.accepted/CIP2015-06-24-call-procedures.adoc [hunk 3]
```diff
@@ -137,10 +137,12 @@ In this case, the trailing `RETURN` clause is omitted and all projected fields a
 
 Procedures may be called standalone either using explicitly passed arguments or using implicitly passed arguments constructed from statement parameters.
 
-The `YIELD` subclause may only be omitted in the standalone form of `CALL` to call a procedure that does not return any result fields.
-In this case the query will return as many (empty) rows as produced by the called procedure.
+The `YIELD` subclause may be omitted in the standalone form of `CALL`. 
+In this case all result fields produced by the procedure implementation are returned by the statement.
+For procedure that does not return any result fields, the query will return as many (empty) rows as produced by the called procedure.
```
Comment (Mats-SX) [inline cip/1.accepted/CIP2015-06-24-call-procedures.adoc:142]:
```suggestion
For a procedure that does not return any result fields, the query will return as many (empty) rows as produced by the called procedure.
```

File: cip/1.accepted/CIP2015-06-24-call-procedures.adoc [hunk 4]
```diff
@@ -137,10 +137,12 @@ In this case, the trailing `RETURN` clause is omitted and all projected fields a
...
 
-A further simplification allowed in the standalone form is to use `YIELD *` to denote that all non-deprecated result fields produced by the procedure implementation are returned by the statement.
+Another subclause allowed in the standalone form is `YIELD *`, which denotes that all result fields produced by the procedure implementation are returned by the statement.
+For procedure that does not return any result fields, the query will return as many (empty) rows as produced by the called procedure.
```
Comment (Mats-SX) [inline cip/1.accepted/CIP2015-06-24-call-procedures.adoc:145]:
```suggestion
For a procedure that does not return any result fields, the query will return as many (empty) rows as produced by the called procedure.
```

File: cip/1.accepted/CIP2015-06-24-call-procedures.adoc [hunk 5]
```diff
@@ -137,10 +137,12 @@ In this case, the trailing `RETURN` clause is omitted and all projected fields a
...
 Procedures may be called standalone either using explicitly passed arguments or using implicitly passed arguments constructed from statement parameters.
 
-The `YIELD` subclause may only be omitted in the standalone form of `CALL` to call a procedure that does not return any result fields.
-In this case the query will return as many (empty) rows as produced by the called procedure.
+The `YIELD` subclause may be omitted in the standalone form of `CALL`. 
+In this case all result fields produced by the procedure implementation are returned by the statement.
+For procedure that does not return any result fields, the query will return as many (empty) rows as produced by the called procedure.
 
-A further simplification allowed in the standalone form is to use `YIELD *` to denote that all non-deprecated result fields produced by the procedure implementation are returned by the statement.
+Another subclause allowed in the standalone form is `YIELD *`, which denotes that all result fields produced by the procedure implementation are returned by the statement.
```
Comment (Mats-SX) [inline cip/1.accepted/CIP2015-06-24-call-procedures.adoc:144]:
```suggestion
A special argument `*` (star) may be given to the `YIELD` subclause in the standalone form.
This denotes that all result fields produced by the procedure implementation are returned by the statement.
```

File: cip/1.accepted/CIP2015-06-24-call-procedures.adoc [hunk 6]
```diff
@@ -190,31 +192,32 @@ RETURN *
...
 [source, cypher]
 ----
 CALL myProc YIELD *
 ----
 
-==== Standalone call to a procedure that does not return result fields
+===== Calling with implicitly passed arguments (parameters) and omitting `YIELD`
 
-Standalone calls without `YIELD` are only supported for procedures that do not return result fields:
+As a further simplification, the `YIELD` subcluase can be omitted alltogether from a standalone procedure call, leading to an even more concise syntax:
```
Comment (Mats-SX) [inline cip/1.accepted/CIP2015-06-24-call-procedures.adoc:205]:
```suggestion
As a further simplification, the `YIELD` subclause can be omitted altogether in a standalone procedure call, leading to an even more concise syntax:
```

**Review by Mats-SX — CHANGES_REQUESTED** [review]
I had some minor suggestions for typos and clarity.

**Review by Hunterness — APPROVED** [review]

**Review by Mats-SX — APPROVED** [review]

**Pull request merged**